src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+{% endDetails %}
+
+## Content surfaced outside of app
+
+The Podcasts app on iOS can surface content in other locations than the actual application, for example, in the system's Widgets view, or in the form of a Siri Suggestion. Having proactive, usage-based calls-to-action that just require a tap to interact with can greatly increase the re-engagement rate of an app like Podcasts.
+
+<figure class="w-figure">
+  <img src="./image6.png" alt="iOS Widget view showing the Podcasts app suggesting a new episode of a podcast." width="300">
+  <figcaption class="w-figcaption">App content is surfaced outside of the main Podcasts app.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/content-indexing-api/">Content Index API</a> allows your application
+  to tell the browser which content of the PWA is available offline.
+  This allows the browser to surface this content outside of the main app.
+  By marking up interesting content in your app as suitable for <a href="https://developers.google.com/search/docs/data-types/speakable">speakable</a>
+  audio playback and by using <a href="https://developers.google.com/search/docs/guides/search-gallery">structured markup</a> in general,
+  you can help search engines and virtual assistants like the Google Assistant present your offerings in an ideal light.
+{% endDetails %}
+
+## Lock screen media control widget
+
+When a podcast episode is playing, the Podcasts app shows a beautiful control widget on the lock screen that features metadata like the episode artwork, the episode title, and the podcast name.
+
+<figure class="w-figure">
+  <img src="./image17.png" alt="iOS media playback widget on the lock screen showing a podcast episode with rich metadata." width="300">
+  <figcaption class="w-figcaption">Media playing in the app can be controlled from the lock screen.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/media-session/">Media Session API</a> lets you specify metadata like artwork, track titles, etc.
+  that then gets displayed on the lock screen, smartwatches, or other media widgets in the browser.
+{% endDetails %}
+
+## Push notifications
+
+Push notifications have become a bit of an annoyance on the web
+(albeit [notification prompts are a lot quieter](https://blog.chromium.org/2020/01/introducing-quieter-permission-ui-for.html) now),
+but if used properly, they can add a lot of value.
+For example, the iOS Podcasts app can optionally notify me of new episodes of podcasts I am subscribed to or recommend me new ones, as well as alert me of new app features.
+
+<figure class="w-figure">
+  <img src="./image12.png" alt="iOS Podcasts app in the 'Notifications' settings screen showing the 'New Episodes' notifications toggle activated." width="300">
```

Comment:
```suggestion
  <img src="./image12.png" class="w-screenshot w-screenshot--filled" alt="iOS Podcasts app in the 'Notifications' settings screen showing the 'New Episodes' notifications toggle activated." width="300">
```

Reply:
This uses a device frame, no screenshot class needed.